diff --git a/src/config/utils/contrail-status.py b/src/config/utils/contrail-status.py
index a75d0f4..2ad8cfb 100755
--- a/controller/src/config/utils/contrail-status.py
+++ b/controller/src/config/utils/contrail-status.py
@@ -366,12 +366,6 @@ def supervisor_status(nodetype, options):
         print ""
         print "== Contrail Supervisor Database =="
         check_status('supervisor-database', options)
-    elif nodetype == 'webui':
-        print "== Contrail Web UI =="
-        check_status('supervisor-webui', options)
-    elif nodetype == 'support-service':
-        print "== Contrail Support Services =="
-        check_status('supervisor-support-service', options)
 
 def package_installed(pkg):
     if distribution == 'debian':

