diff --git a/src/config/utils/contrail-status.py b/src/config/utils/contrail-status.py
index 55dbe90..6523bf2 100755
--- a/src/config/utils/contrail-status.py
+++ b/src/config/utils/contrail-status.py
@@ -370,13 +370,6 @@ def supervisor_status(nodetype, options):
         print ""
         print "== Contrail Supervisor Database =="
         check_status('supervisor-database', options)
-    elif nodetype == 'webui':
-        print "== Contrail Web UI =="
-        check_status('supervisor-webui', options)
-    elif nodetype == 'support-service' and \
-         distribution == 'debian':
-        print "== Contrail Support Services =="
-        check_status('supervisor-support-service', options)
 
 def package_installed(pkg):
     if distribution == 'debian':
-- 
2.1.4
